common/djangoapps/third_party_auth/models.py

```diff
@@ -692,7 +699,7 @@ def get_config(self):
             conf = {}
         attrs = (
             'attr_user_permanent_id', 'attr_full_name', 'attr_first_name',
-            'attr_last_name', 'attr_username', 'attr_email', 'entity_id')
+            'attr_last_name', 'attr_username', 'attr_email', 'entity_id', 'country')
```

Comment:
fun fact: while digging through this for my own issue, I found why we don't include everything here.
The ProviderConfig object contains a bunch of values (such as booleans like "send_to_registration_first") that have nothing to do with the SAML mapping process. Those values are still saved into the object, but this "conf" we're building here is purely SAML related values.

Reply:
That actually answers a couple questions I had for you in terms of the frontend admin portal lol

Reply:
The frontend admin portal includes even less options, but that is because we are relying 100% on the backend to fill those in. There are certain options where we don't want users to fill in the values and we just the same values anyways, such as I think backend or backend-name (it's tpa-saml) as well as a bunch of these booleans. If you have questions about the flow from frontend to backend or why we left out a bunch in the frontend, feel free to ask outside this.

But this finding was more directly related to trying to understand re-construction of a samlproviderconfig object and how it's used and realizing we use the non-directly-saml related attributes of that object elsewhere in logic gates throughout the third_party_auth app and logistration.